payments-core/src/test/java/com/stripe/android/view/CardInputWidgetTest.kt

```diff
@@ -1774,8 +1774,9 @@ internal class CardInputWidgetTest {
     fun `Re-fetches card brands when first eight are deleted and re-entered`() = runCardInputWidgetTest(
         true
     ) {
-        cardNumberEditText.setText("4000 0025 0000 1001")
-        assertThat(cardBrandView.possibleBrands.size).isEqualTo(2)
+        // TODO(tjclawson): Figure out fix for entering and deleting same card number
```

Comment:
Can you remove the TODO and create a jira?

This is a Jay pet peeve. We don't track TODOs, but we track jiras.

Reply:
Done 👍  https://jira.corp.stripe.com/browse/RUN_MOBILESDK-3546